### DISCUSSION TITLE
Submenu?

### DISCUSSION THREAD

**Discussion by matt-dale** [chakra-ui/chakra-ui#3671] [Q&A] [answered]
Has anyone built a nice submenu yet? I saw this [example on Twitter](https://twitter.com/thesegunadebayo/status/1257280339963269121) which looks beautiful and I'm wondering if it will become a core component or if we should try to build it custom?

**Comment by with-heart** [comment, marked as answer]
We had this at one, but due to various concerns around complexity and performance (I think?) we removed it with the intention of revisiting it someday. That day has not yet come. 😅
